NitroxClient/Communication/Packets/Processors/DiscordRequestIPProcessor.cs

```diff
@@ -1,13 +1,33 @@
-﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+﻿using System.Net;
+using System.Threading.Tasks;
+using NitroxClient.Communication.Packets.Processors.Abstract;
 using NitroxClient.MonoBehaviours.Discord;
+using NitroxModel.Helper;
 using NitroxModel.Packets;
 
 namespace NitroxClient.Communication.Packets.Processors;
 
 public class DiscordRequestIPProcessor : ClientPacketProcessor<DiscordRequestIP>
 {
-    public override void Process(DiscordRequestIP packet)
+    public override async void Process(DiscordRequestIP packet)
     {
-        DiscordClient.UpdateIpPort(packet.IpPort);
+        string ipPort = await GetIpPortAsync();
+        DiscordClient.UpdateIpPort(ipPort);
+    }
+
+    /** Get the WAN IP address or the Hamachi IP address if the WAN IP address is not available. */
```

Comment:
Pls no java style code docs, these aren't parsed in IDEs or doc export tools.

```suggestion
    /// <summary>
    ///     Get the WAN IP address or the Hamachi IP address if the WAN IP address is not available.
    /// </summary>
```